mir-module/src/main/resources/config/mir/messages_en.properties

```diff
@@ -864,3 +864,4 @@ subselect.category.cancel = Cancel
 subselect.category.title  = Choose category
 
 user.profile.id.orcid = Your ORCID iD
+mir.register.username.placeholder = maximum of 16 characters, allowed characters are a-z, 0-9 and ._-
```

Comment:
New entry should be at the proper location in the file, just as in the German version.

Reply:
That is just academic. The line number still don't match since the message properties files are topsy turvy. They should be sorted.
I changed the location anyway.